iam-policy-autopilot-lints/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "node_kind_literal"
+version = "0.1.0"
+authors = ["IAM Policy Autopilot Contributors"]
+description = "Dylint lint to enforce usage of node_kinds constants instead of string literals"
+edition = "2021"
+publish = false
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+clippy_utils = { git = "https://github.com/rust-lang/rust-clippy", rev = "20ce69b9a63bcd2756cd906fe0964d1e901e042a" }
```

Comment:
Is there a reason for this particular version? Doesn't seem to be the latest one. Could you add a comment. I see that this is not published as a crate.

Reply:
No reason other than it builds with the nightly compiler version we're using. Added a comment about this.